webpack.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  entry: {
+    app: './static/js/src/app.js',
+    'blog-feed': './static/js/src/blog-feed.js',
+  },
+  mode: process.env.ENVIRONMENT === "devel" ? "development" : "production",
```

Comment:
Why don't we just use 'development' and 'production'?

Reply:
I think it's just a discrepancy between what node sets and what webpack expects. This is the same for all our projects I think.